pkg/service/service.go

```diff
@@ -219,6 +224,84 @@ func ListWithDetailedStatus(client *occlient.Client, applicationName string) (Se
 	}, nil
 }
 
+// ListOperatorServices lists all operator backed services
+func ListOperatorServices(client *kclient.Client) error {
+	glog.V(4).Info("Getting list of services")
+
+	// First let's get the list of all the operators in the namespace
+	csvs, err := client.GetClusterServiceVersionList()
+	if err != nil {
+		return errors.Wrap(err, "Unable to list operator backed services")
+	}
+
+	var allCRInstances []unstructured.Unstructured
+
+	// let's get the Services a.k.a Custom Resources (CR) defined by each operator, one by one
+	for _, csv := range csvs.Items {
+		glog.V(4).Infof("Getting services started from operator: %s\n", csv.Name)
+		customResources := client.GetCustomResourcesFromCSV(csv)
+
+		// list and write active instances of each service/CR
+		instances, err := getCRInstances(client, customResources)
+		if err != nil {
+			return err
+		}
+
+		// assuming there are more than one instances of a CR
+		allCRInstances = append(allCRInstances, instances...)
+	}
+
+	displayServices(allCRInstances)
+
+	return nil
+}
+
+func getGVRFromCR(cr olm.CRDDescription) (group, version, resource string) {
+	version = cr.Version
+
+	gr := strings.SplitN(cr.Name, ".", 2)
+	resource = gr[0]
+	group = gr[1]
+
+	return
+}
+
+func getCRInstances(client *kclient.Client, customResources []olm.CRDDescription) ([]unstructured.Unstructured, error) {
+	var instances []unstructured.Unstructured
+
+	for _, cr := range customResources {
+		group, version, resource := getGVRFromCR(cr)
+		glog.V(4).Infof("Getting instances of: %s\n", cr.Name)
+		list, err := client.ListDynamicResource(group, version, resource)
+		if err != nil {
+			return []unstructured.Unstructured{}, err
+		}
+
+		if len(list.Items) > 0 {
+			instances = append(instances, list.Items...)
+		}
+	}
+	return instances, nil
+}
+
+func displayServices(list []unstructured.Unstructured) {
```

Comment:
we shouldn't display here and instead display in the CLI package

Reply:
Done. Thanks for pointing. :+1: